### ISSUE TITLE
fix for strict_indent do not work for file

### ISSUE THREAD

**Issue by zombiedk** [puppetlabs/puppet-lint#109]
when you use `puppet-lint --fix file.pp` to correct indentation it do not make any changes to file sektion

Input
```
  file { '/etc/ssh/sshrc':
mode         => 'a+rx',
    source => "puppet:///modules/profile/auth/ssh/sshrc",
}
```
expected output
```
  file { '/etc/ssh/sshrc':
    mode   => 'a+rx',
    source => "puppet:///modules/profile/auth/ssh/sshrc",
  }
```

## Environment
 - Version [3.3.0]
 - Platform [Ubuntu 22.10]

**Comment by ekohl** [comment]
The strict_indent plugin is here: https://github.com/voxpupuli/puppet-lint-strict_indent-check

**Comment by zombiedk** [comment]
thanks have create an issue there
[fix for strict_indent do not work for file #32](https://github.com/voxpupuli/puppet-lint-strict_indent-check/issues/32)
